src/lomc.jl

```diff
@@ -267,7 +267,7 @@ function lomc!(ham, v; df=DataFrame(), name="lomc!", kwargs...)
     return lomc!(state, df; name)
 end
 function lomc!(ham; style=IsStochasticInteger(), kwargs...)
-    v = DVec(starting_address(ham)=>10; style)
+    v = PDVec(starting_address(ham) => 10; style)
```

Comment:
Why change the default to `PDVec`? Note that the docstring still mentions `DVec` as the default